domains/Zainab.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "username": "zainaBinteAzhar",
+    "email": "164602326+zainabBinteAzhar@users.noreply.github.com"
```

Invalid Email, if you dont want to share your email, you are free to remove this
```suggestion
    "username": "zainaBinteAzhar"
```